src/util/CookieUtil.js

```diff
@@ -54,5 +55,26 @@ define(['okta', 'vendor/plugins/jquery.cookie'], function (Okta) {
     removeCookie(REMEMBER_DEVICE_COOKIE_NAME);
   };
 
+  fn.isAutoPushEnabled = function (username) {
+    if (username === undefined) {
+      return false;
+    }
+    return $.cookie(AUTO_PUSH_COOKIE_PREFIX + CryptoUtil.getStringHash(username)) === "true";
+  };
+
+  fn.setAutoPushCookie = function (username) {
+    if (username === undefined) {
+      return;
+    }
+    setCookie(AUTO_PUSH_COOKIE_PREFIX + CryptoUtil.getStringHash(username), true);
```

Comment:
Let's create a private helper function to generate the cookie name (i.e. don't have the same logic in two places, which can break if someone just updates one in the future).